monors.py

```diff
@@ -247,6 +247,8 @@ def main():
     pulls = gh.repos (cfg["owner"]) (cfg["repo"]).pulls ().get (state="open", sort="updated", direction="desc", per_page = 100)
     logging.info("found %d pull requests", len (pulls))
 
+    # We have a limit on the number of HTTP request to the github API (5000/h), each PR takes 3+ request, every 5 minutes.
+    # With 25, we arrived to ~2000/h requests per hour, so with 50, we should arrive at ~4000/h.
```

Comment:
Can we instead just process the PRs that have "@monojenkins merge"? We should be able to do this via a simple search, like https://github.com/mono/mono/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+%22%40monojenkins+merge%22